README.md

```diff
@@ -33,17 +33,85 @@ The protocol is typically run using the `imp_sampcon` command line tool:
    sampling was exhaustive.
 
 For a full demonstration of the protocol, see its usage in
-IMP's [actin modeling tutorial](https://integrativemodeling.org/tutorials/actin/analysis.html).
+IMP's [actin modeling tutorial](https://integrativemodeling.org/tutorials/actin/analysis.html) and [RNA Polymerase 2 tutorial](https://github.com/salilab/imp_analysis_tutorial).
```

Comment:
I'm not sure it's a good idea to link to this tutorial since Daniel never finished it.